socorro/unittest/external/es/test_supersearch.py

```diff
@@ -1057,6 +1061,20 @@ def test_get_with_sub_aggregations(self):
         version_sub_facet = res['facets']['signature'][0]['facets']['version']
         eq_(len(version_sub_facet), number_of_crashes)
 
+        # Test with a third level aggregation.
+        kwargs = {
+            '_aggs.product.version': ['_cardinality.signature'],
+        }
+        res = self.api.get(**kwargs)
+
```

Comment:
That would simply be ignored. I was actually wondering, should SuperSearch be more aggressive in validating arguments? At the moment it simply just ignores stuff that it cannot use (like incorrectly spelled field names, or incorrect parameters), and returns some results that might not be what you were expecting. Should it instead raise exceptions when something seems off?